windows/win_iis_webapppool.ps1

```diff
@@ -90,10 +90,14 @@ try {
       Stop-WebAppPool -Name $name -ErrorAction Stop
       $result.changed = $TRUE
     }
-    if ((($state -eq 'started') -and ($pool.State -eq 'Stopped')) -or ($state -eq 'restarted')) {
+    if ((($state -eq 'started') -and ($pool.State -eq 'Stopped'))) {
       Start-WebAppPool -Name $name -ErrorAction Stop
       $result.changed = $TRUE
     }
+    if ($state -eq 'restarted') {
+      Restart-WebAppPool -Name $name
```

Comment:
Gotcha, please check out the latest commit which starts the pool instead of recycling it when state is state to restarted while the pool is stopped